tools/travis-before-install.sh

```diff
@@ -33,5 +36,11 @@ python -V
 popd
 
 pip install --upgrade pip setuptools
+# pycparser 2.19 doesn't install correctly when Python's optimization level is
+# set to strip docstrings (see https://github.com/eliben/pycparser/issues/291),
+# so preemptively install pycparser with PYTHONOPTIMIZE unset.  If a future
+# version of pycparser doesn't have this problem, the following line can be
+# removed.
```

Comment:
I think the comment should mention that pycparser does not ship a wheel, so it must be built and building fails with PYTHONOPTIMIZE=2